packages/tx/README.md

```diff
@@ -364,6 +364,7 @@ The following L2 networks have been tested to work with `@ethereumjs/tx`, see us
 | xDai Chain               |  `Common.xDaiChain`                   |  [#1323](https://github.com/ethereumjs/ethereumjs-monorepo/issues/1323) |
 | Optimistic Kovan         | `Common.OptimisticKovan`              | [#1554](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1554)    |
 | Optimistic Ethereum      | `Common.OptimisticEthereum`           | [#1554](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1554)    |
+| Polygon Amoy Testnet     | `CustomChain.PolygonAmoy`             | [#3379](https://github.com/ethereumjs/ethereumjs-monorepo/issues/3379)  |
```

Comment:
Did you test and confirm that transactions created with our libraries are accepted and included in Amoy blocks?